components/brave_wallet/browser/keyring_controller.h

```diff
@@ -278,7 +311,7 @@ class KeyringController : public KeyedService, public mojom::KeyringController {
   std::unique_ptr<HDKeyring> default_keyring_;
   std::unique_ptr<base::OneShotTimer> auto_lock_timer_;
   std::unique_ptr<PrefChangeRegistrar> pref_change_registrar_;
-
+  std::unique_ptr<FilecoinKeyring> filecoin_keyring_;
```

Comment:
If we need to support multiple keyrings, we need a map keyed by keyring id.
Currently, your change makes keyring id other than filecoin all return default_keyring_ which is not correct

Reply:
this is done in my `filecoin-19285` branch, there are other changes for keyring controller for multi-currencies support, I will create PR right after this one

Reply:
ok, please link the PR here when you create it.

Reply:
https://github.com/brave/brave-core/pull/11687 linking PR with multiple keyrings